python/lsst/ts/m2com/mock/mock_control_open_loop.py

```diff
@@ -0,0 +1,445 @@
+# This file is part of ts_m2com.
+#
+# Developed for the Vera Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["MockControlOpenLoop"]
+
+import numpy as np
+import pandas as pd
+
+from ..constant import NUM_ACTUATOR, NUM_TANGENT_LINK, MIRROR_WEIGHT_KG
+from ..enum import ActuatorDisplacementUnit
+
+
+class MockControlOpenLoop:
+    """Mock open-loop control.
+
+    Attributes
+    ----------
+    inclinometer_angle : `float`
+        Inclinometer angle in degree.
+    open_loop_max_limit_is_enabled : `bool`
+        The maximum limit of open-loop control is enabled or not.
+    is_running : `bool`
+        The script is running or not.
+    actuator_steps : `numpy.ndarray [int]`
+        Actuator steps.
+    """
+
+    # 1 step equals 1.9967536601e-5 millimeter
+    # In the simulation, we just use a single value. In the real system, each
+    # actuator has its own calibrated value.
+    STEP_TO_MM = 1.9967536601e-5
+
+    # Limits of force in Newton
+    LIMIT_FORCE_AXIAL = 489.3  # 110 lbf
+    LIMIT_FORCE_TANGENT = 6005.1  # 1350 lbf
+
+    # Maximum limits of force in Newton
+    MAX_LIMIT_FORCE_AXIAL = 622.75  # 140 lbf
+    MAX_LIMIT_FORCE_TANGENT = 6227.51  # 1400 lbf
+
+    def __init__(self):
+
+        self.inclinometer_angle = 0
+        self.open_loop_max_limit_is_enabled = False
+        self.is_running = False
+
+        # Static transfer matrix used to simulate the delta force from the
+        # change of steps
+        self._static_transfer_matrix = np.array([])
+
+        self.actuator_steps = np.zeros(NUM_ACTUATOR, dtype=int)
+
+        # Selected actuator IDs to do the movement
+        self._selected_actuators = list()
+
+        # Displacement of steps to do the movement
+        self._displacement_steps = 0
+
+    def read_file_static_transfer_matrix(self, filepath, skiprows=7):
```

Comment:
I tried this (after adding the `#` manually) and got the following error:

```
ParserError: Error tokenizing data. C error: Expected 2 fields in line 9, saw 3
```

This is because the original file by Harris does not provide the correct format of `csv` file. For example, there is the problem of inconsistent delimiter as well. Another way I tried is to use the `numpy` to read the file (to workaround the inconsistent delimiter). But this will need to assign which column to read and use as well. Therefore, after some test, I decide to keep my original use.

I will discuss with John to change all the configuration files to be `yaml` or `json` in a latter time. After that, I will change all the configuration files into the decided file format. This should solve the problem at that time. And this might be helpful when I begin to update the interface of M2 cell LabVIEW code. Thanks!